examples/gno.land/r/gnoland/faucet/admin.gno

```diff
@@ -5,44 +5,77 @@ import (
 	"std"
 )
 
-func AdminSetInPause(inPause bool) error {
+func AdminSetInPause(inPause bool) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gInPause = inPause
-	return nil
+	return ""
 }
 
-func AdminSetMessage(message string) error {
+func AdminSetMessage(message string) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gMessage = message
-	return nil
+	return ""
 }
 
-func AdminSetPerTransferSend(send std.Coins) error {
+func AdminSetTransferLimit(amount int64) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
-	gPerTransferSend = send
-	return nil
+	gLimit = std.Coin{Denom: "ugnot", Amount: amount}
+	return ""
 }
 
-func AdminSetAdminAddr(addr std.Address) error {
+func AdminSetAdminAddr(addr std.Address) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gAdminAddr = addr
-	return nil
+	return ""
 }
 
-func AdminSetControllerAddr(addr std.Address) error {
+func AdminAddController(addr std.Address) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
-	gControllerAddr = addr
-	return nil
+
+	size := gControllers.Size()
+
+	if size >= gControllersSize {
+		return "can not add more controllers than allowed"
+	}
+
+	if gControllers.Has(addr.String()) {
+		return addr.String() + " exists, no need to add."
+	}
```

Comment:
This is probably just my style preference, but what do you think about this:

* Define local (unexported helper checkers) that return an error (or string, as I see errors are not really supported)
* Simply call them within this method, instead of laying them out

So it would look something like this:

```go
// ...
if err := controllerPresent(addr.String()); err != "" {
    return err
}
// ...
```
where `controllerPresent` of course being this `g.Controllers.Has` check.

What do you think?

Reply:
What are we getting with additional abstractions here?

Reply:
Just simpler top level code reads, maybe it's just a personal preference. I'm resolving this convo because it's not critical